integration-tests/integration/globalsearch/globalSearch.spec.js

```diff
@@ -0,0 +1,192 @@
+    })
+  })
+
+  it('should present global search results with outside descriptions', () => {
+    cy.task('stubGlobalSearchMultiplePages')
+    cy.task('stubOffenderMovements')
+    cy.visit('/global-search-results?searchText=common')
+    const globalSearchPage = GlobalSearchPage.verifyOnResultsPage()
+    globalSearchPage.rows().should(rows => {
+      expect(rows).to.have.lengthOf(21)
+      expect(rows[1]).to.have.text(
+        'Common, Fred1T1001AA15/10/1977Outside - released from Low Newton (HMP)Common, Fred1'
+      )
+    })
+  })
+
+  it('should be able to navigate pages of results', () => {
+    cy.task('stubGlobalSearchMultiplePages')
+    cy.task('stubOffenderMovements')
+    cy.visit('/global-search-results?searchText=common')
+    const globalSearchPage = GlobalSearchPage.verifyOnResultsPage()
+    globalSearchPage.rows().should(rows => {
+      expect(rows).to.have.lengthOf(21)
+      expect(rows[20]).to.have.text('Common, Fred20T1020AA15/10/1977Leeds HMPCommon, Fred20')
+    })
+    globalSearchPage.nextPage().click()
+    globalSearchPage.rows().should(rows => {
+      expect(rows).to.have.lengthOf(2)
+      expect(rows[1]).to.have.text('Common, Fred21T1021AA15/10/1977Leeds HMPCommon, Fred21')
+    })
+    globalSearchPage.previousPage().click()
+    globalSearchPage.rows().should(rows => {
+      expect(rows).to.have.lengthOf(21)
+      expect(rows[20]).to.have.text('Common, Fred20T1020AA15/10/1977Leeds HMPCommon, Fred20')
+    })
+  })
+
+  it('should present search filters', () => {
+    cy.task('stubGlobalSearch')
+    cy.task('stubOffenderMovements')
+    cy.visit('/global-search-results?searchText=smith, john')
+    const globalSearchPage = GlobalSearchPage.verifyOnResultsPage()
+    globalSearchPage
+      .showFilters()
+      .should('have.text', 'Show filters')
+      .click()
+      .should('have.text', 'Hide filters')
+    globalSearchPage
+      .locationSelect()
+      .should('have.value', 'ALL')
+      .select('OUT')
+    globalSearchPage
+      .genderSelect()
+      .should('have.value', 'ALL')
+      .select('F')
+    globalSearchPage.dobDay().type('1')
+    globalSearchPage.dobMonth().type('1')
+    globalSearchPage.dobYear().type('1970')
+
+    globalSearchPage
+      .form()
+      .submitButton()
+      .click()
+
+    globalSearchPage.clearFilters().click()
+    globalSearchPage.locationSelect().should('have.value', 'ALL')
+    globalSearchPage.genderSelect().should('have.value', 'ALL')
+    globalSearchPage.dobDay().should('have.value', '')
+    globalSearchPage.dobMonth().should('have.value', '')
+    globalSearchPage.dobYear().should('have.value', '')
+
+    globalSearchPage
+      .showFilters()
+      .click()
+      .should('have.text', 'Show filters')
+
+    cy.task('verifyGlobalSearch').should(requests => {
+      expect(requests).to.have.lengthOf(2)
+      expect(JSON.parse(requests[1].body)).to.deep.equal({
```

Comment:
this is quite nice - actually inspect wiremock requests to ensure that correct parameters sent to prisoner offender search